<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDC</class>
 <widget class="QWidget" name="QDC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel: </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_channelN">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_qdc_enable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>on/off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold: </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_threshold">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>125</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4080</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set it</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>20</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearChannel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear channel</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="qdc_profile" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>nBins: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>590</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_nbins">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of events:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_nevents">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>415</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qdc_startprofile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stopprofile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_qdc_vmin">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>535</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_qdc_vmax">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <double>100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>4000.000000000000000</double>
   </property>
   <property name="value">
    <double>400.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearProfile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>125</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>*0.1 (pC)</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
